### ISSUE TITLE
Updated to 2.3.2 and it no longer opens.

### ISSUE THREAD

**Issue by Julioevm** [alainm23/planify#322]
**Describe the bug**
After updating to 2.3.3 the program starts, you can see the process running, but the interface doesnt load up.
Tried to re-install but it didn't help.

**Expected behavior**
The program should show up when you run it.

**Screenshots**
![image](https://user-images.githubusercontent.com/38758281/80186606-89ee9580-860e-11ea-8c9f-cecbe2e173be.png)

**Desktop (please complete the following information):**
 - OS or DE: pantheon
 - Version 5.1.3

**Additional context**
It worked without issues prior to the update.

**Comment by 0xhtml** [comment]
I got the same issue. I tried resetting the settings and the database but both didn't help. Then I tried running Planner from the terminal and got to the following results:
- If you just start it, it starts in the background and doesn't show up.
- If you start it with `com.github.alainm23.planner --help` it show an error:
    ```
    GDBus.Error:org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying
    ```
- But if you start it now (after the error from `--help`) by running `com.github.alainm23.planner` it starts and shows up again.

I got some errors/warnings while running Planner like above and I'm not sure if they are normal or not, so there you go:
```
(com.github.alainm23.planner:11515): Gtk-CRITICAL **: 11:49:36.038: gtk_box_pack: assertion '_gtk_widget_get_parent (child) == NULL' failed

(com.github.alainm23.planner:11515): Gtk-CRITICAL **: 11:49:36.085: _gtk_widget_get_preferred_size_for_size: assertion 'size >= -1' failed

** (com.github.alainm23.planner:11515): CRITICAL **: 11:49:36.130: SimpleCommand.vala:103: Couldn't launch command echo $DESKTOP_SESSION in the directory .: Child process "echo" could not be executed (file or directory not found)

(com.github.alainm23.planner:11515): GLib-CRITICAL **: 11:49:36.130: g_child_watch_add_full: assertion 'pid > 0' failed

(com.github.alainm23.planner:11515): Gtk-CRITICAL **: 11:49:36.190: _gtk_widget_get_preferred_size_for_size: assertion 'size >= -1' failed
```

**Comment by edwinvandeven** [comment]
Can confirm, I'm experiencing the same behavior. It doesn't load after updating by clicking the icon and I'm getting the same messages mentioned my @0xhtml .

As a side note to the developer;
Keep up the awesome work! I really love this application, you've created something great!

**Comment by alainm23** [comment]
Thank you very much, the error has already been reported and solved, please wait for update 2.3.3 that still needs to be approved by AppCenter.
